.github/workflows/update-url.yml

```diff
@@ -0,0 +1,16 @@
+name: Update Download URL
+
+on:
+  release:
+    types: [published, prereleased]
```

Comment:
I've been using `prereleased` to test this action running on a prerelease but we'll want to remove this